pipeline-modules/deployment-service/aws/lambda-gateway.yaml

```diff
@@ -90,7 +90,7 @@ inputs:
     - app_env
     - repository
     - tag
-    - application
```

Comment:
Is applicationName & application internal variable hold the same value or a different value.

Reply:
I observed that we're not setting any internal var as `application` for deployment pipelines, and the intent to use the name seems to be application name only. So it led me to update the key name here.